src/twisted/python/test/test_deprecate.py

```diff
@@ -732,6 +733,15 @@ def someProperty(self, value):
 
 
 
+@deprecatedKeywordParameter(Version('Twisted', 19, 2, 0), 'foo')
+def functionWithDeprecatedParameter(a, b, c=1, foo=2, bar=3):
+    """
+    Function with a deprecated keyword parameter.
+    """
+    pass
```

Comment:
Not a big deal, but if you have a valid doc string, the function is still valid
if you omit the `pass`.